Automattic/woocommerce-payments
# WCPay overview - display fees per payment method

Fixes #2377

#### Changes proposed in this Pull Request

This PR changes the way that fees are displayed on the WCPay overview page, under the "Account details" section. 
The fees were displayed ony for the card payment method, but since new payment methods have been added, having a different base and promo fees, we need to display only the discounts based on payment method.
So, the purpose is to replace the "base fee" (the standard card transaction fee) information with a new section "Active discounts" .


Overview page before changes:
<img width="781" alt="Screenshot 2021-10-13 at 08 49 09" src="https://user-images.githubusercontent.com/82826872/137074649-19fcb871-6a1e-45dc-8b54-cfbb3f3b816b.png">



Overview page after changes:

<img width="708" alt="Screenshot 2021-10-13 at 02 26 24" src="https://user-images.githubusercontent.com/82826872/137073422-bcdf2206-3acf-439d-a72e-43860c7f953d.png">

Design: 35wcSfexs3z8pxrj4daJnH-fi-0%3A1 









#### Testing instructions
- Add a discount fee with multiple payment methods. You can do this in 2 ways:
1. Select a fee structure on server side (wcpay/fees folder) and add it using the CLI command: 
`wp wcpay add_fee_structure_to_account <stripe_account_id> --fee_structure_id=<fee_structure_id>`
2. Add the fee manually in the database,`wcpay_test_account_fee_structures` table. In order to have the fee active `end_date` field should be in the future or null.

We should have only 1 fee structure active for an account. In theory, an account can have more than one account fee structure at the same time. However, the current implementation of promotions only allows a single fee structure to be applied at the point an account is created. So we're unlikely to see this in practice at the moment.

- Re-login or clear the account cache 
- Go to Payments -> Overview
- Check if the discount is displayed properly under the Account details->Active discounts  section for each payment method






-------------------


**Post merge**
